include/spdlog/sinks/dist_sink.h

```diff
@@ -0,0 +1,71 @@
+//
+// Copyright (c) 2015 David Schury, Gabi Melman
```

Comment:
Believe we need to include David Schury in the LICENSE in addition to Gabi Melman: https://github.com/apache/nifi-minifi-cpp/blob/ff7aa7fc3bf55681398427f1de6af7f352a100fa/LICENSE#L243